client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -85,6 +95,41 @@ module.exports = React.createClass( {
 		return info;
 	},
 
+	handleDomainVolumeSelection: function( event ) {
+		event.preventDefault();
+		let volume = parseInt( event.target.value );
```

Comment:
it looks like we can use `const` here since `volume` isn't assigned anything after the initial value